components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/internal/impl/UserSessionManagementServiceImpl.java

```diff
@@ -150,6 +153,8 @@ private void terminateSessionsOfUser(List<String> sessionList) {
 
         for (String session : sessionList) {
             sessionManagementService.removeSession(session);
+            addAuditLogs(session, FrameworkUtils.getLoginTenantDomainFromContext(),
```

Comment:
for terminate audit logs, shall we add to https://github.com/wso2/carbon-identity-framework/blob/50552a805f2b03c7ffe8457469376b0dd0fc6f0a/components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/internal/impl/ServerSessionManagementServiceImpl.java#L58 ?